grapple/types/interfaces.py

```diff
@@ -209,3 +209,18 @@ def resolve_raw_value(self, info, **kwargs):
             return self.value.source
 
         return self.value
+
+
+def get_snippet_interface():
+    return import_string(grapple_settings.SNIPPET_INTERFACE)
```

Comment:
I keep circling to the idea that we want the custom interfaces to inherit from our base ones (be it page, snippet or StreamField).

What are the risks of not doing so? e.g. fully replacing the `PageInterface`?

Reply:
> What are the risks of not doing so? e.g. fully replacing the `PageInterface`?

I'm not sure. But, if I'm replacing Grapple's default page/snippet interface, I'm probably inheriting from the default one, because it feels like the right thing to do.

Reply:
We can add a check, and print a warning, I guess? But it feels outside the scope of this ticket.

Reply:
I've created a ticket to address this: https://github.com/torchbox/wagtail-grapple/issues/408.